app/controllers/candidate_interface/references/review_controller.rb

```diff
@@ -136,11 +106,9 @@ def section_complete_params
       def set_references
         if FeatureFlag.active?(:reference_selection)
           @references_selected = current_application.application_references.includes(:application_form).selected
-          @references_given = current_application.application_references.includes(:application_form).feedback_provided_but_not_selected
```

Comment:
Can delete the `ApplicationReference.feedback_provided_but_not_selected` scope as well, as it's no longer used after this change.

Reply:
While we're at it, let's delete the `ApplicationReference.selected` I added earlier as well. Not needed, Rails already creates that scope!